polaris-react/package.json

```diff
@@ -84,6 +85,8 @@
     "@storybook/preview-api": "^7.6.4",
     "@storybook/react": "^7.6.4",
     "@storybook/react-vite": "^7.6.4",
+    "@storybook/theming": "^7.6.4",
+    "@types/js-yaml": "^4.0.5",
```

Comment:
Are these missing deps that _happened_ to be available because of yarn's hoisting, but now need to be explicitly installed per workspace so pnpm can symlink them correctly? If so; nice!

Reply:
That's right! They were being used directly but weren't being installed explicitly, and began failing because pnpm won't hoist them and calls them "ghost dependencies"